C/Makefile

```diff
@@ -1,8 +1,12 @@
 PROGRAM=blurhash
-
+DECODER=blurhash_decoder
 $(PROGRAM): blurhash_stb.c encode.c encode.h stb_image.h
 	$(CC) -o $@ blurhash_stb.c encode.c -lm
 
+decoder: decoder/decode_main.c decoder/decode.c decoder/decode.h decoder/stb_writer.h
+	$(CC) -o $(DECODER) decoder/decode_main.c decoder/decode.c  -Idecoder/ -lm
+
 .PHONY: clean
 clean:
 	rm -f $(PROGRAM)
+	rm -f $(DECODER)
```

Comment:
this should be added to the .gitignore file

Reply:
Yes! Correct,
Added this in `.gitignore'